crates/stackable-versioned-macros/tests/inputs/k8s/pass/conversion_tracking.rs

```diff
@@ -0,0 +1,33 @@
+use kube::CustomResource;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use stackable_versioned::versioned;
+// ---
+#[versioned(
+    version(name = "v1alpha1"),
+    version(name = "v1beta1"),
+    version(name = "v1"),
+    k8s(
+        group = "stackable.tech",
+        status = MyStatus,
+        options(experimental_conversion_tracking),
+    )
+)]
+// ---
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema, CustomResource)]
+pub(crate) struct FooSpec {
+    #[versioned(added(since = "v1beta1"), changed(since = "v1", from_name = "bah"))]
+    bar: usize,
+    baz: bool,
+}
+// ---
+fn main() {}
+
+#[derive(Clone, Debug, JsonSchema, Deserialize, Serialize)]
+pub struct MyStatus {
```

Comment:
nit: Can we please rename the struct to `FooStatus`, now that the name doesn't clash any more?
This better represents how we currently use status in existing operators

Reply:
Good thinking, thank you. Will update accordingly.

Reply:
Done in 46da1d6.